evaluation_scripts/cfid/cfid_metric.py

```diff
@@ -152,7 +152,7 @@ def _get_embed_im(self, multi_coil_inp, mean, std, maps):
 
             S = sp.linop.Multiply((self.args.im_size, self.args.im_size), maps[i])
 
-            im = torch.tensor(S.H * tensor_to_complex_np(unnormal_im.cpu())).abs().cuda()
+            im = torch.real(torch.tensor(S.H * tensor_to_complex_np(unnormal_im.cpu()))).cuda()
```

Comment:
It'd be good to add a comment to the code to explain this modification. Eventually, leave the previous line commented to show the previous implementation.

Also, this function will probably be used in the radio problem, so it'd be good to check with Matthijs what makes more sense to use in his problem. This change could be a potential bug for his application if it goes unnoticed.